static/js/popover_menus.js

```diff
@@ -56,10 +56,25 @@ function on_show_prep(instance) {
     popovers.hide_all_except_sidebars(instance);
 }
 
+function tippy_no_propagation(target, popover_props) {
+    $("body").on("click", target, (e) => {
+        e.preventDefault();
+        e.stopPropagation();
+        const instance = e.currentTarget._tippy;
+        if (instance) {
+            instance.hide();
+            return;
+        }
+        tippy(e.currentTarget, {
+            ...default_popover_props,
+            showOnCreate: true,
+            ...popover_props,
+        });
+    });
+}
```

Comment:
Can you add a comment explaining what precisely this does? I'm a bit confused; is the intent that we just want to avoid additional things happening due to a click that is designed to open/close the popover itself and do nothing else? 

Also, should the name somehow hint that this is designed to implement a toggle, like `delegate_tippy_toggle_popover`?

Reply:
Added this comment:
```
    // Everything we do here can be implemented just using `delegate`like we do for
    // `.compose_mobile_button` popover except `stopPropagation`.
    // So, only use it when you want clicks on the target element to not propagate
    // throughout the app.
```

which also explains why `tippy_no_propagation` is a better name.